packages/store-sync/src/recs/syncToRecs.ts

```diff
@@ -15,6 +19,7 @@ type SyncToRecsOptions<TConfig extends StoreConfig = StoreConfig> = SyncOptions<
 type SyncToRecsResult<TConfig extends StoreConfig = StoreConfig> = SyncResult & {
   components: RecsStorageAdapter<TConfig>["components"];
   stopSync: () => void;
+  getResourceSelector: (functionSelector: Hex) => Promise<Hex>;
```

Comment:
might be worth adding this to `SyncResult` type and filling this in for sqlite/postgres too

Reply:
the current implementation depends on Components which is unique to RECS it seems? Not very familar with the code there yet but looks like for both we can only use the RPC call. Do you mind me handling it as a follow-up?

Reply:
yes, we'd need to add implementations for each sqlite and postgres, but it should be fairly straightforward: we use drizzle for queries and there should be strongly typed schemas for built-in tables like FunctionSelectors

Reply:
Sg!